cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,281 @@ Cask Data Application Platform Release Notes
+
+- `CDAP-4230 <https://issues.cask.co/browse/CDAP-4230>`__ -
+  ETL supports DAG in pipeline, which will allow users to fork and merge in pipeline. ETLConfig has been
+  updated to allow representing DAG.
+
+- `CDAP-3857 <https://issues.cask.co/browse/CDAP-3857>`__ -
+  Added experimental support for running ETL Batch applications on Spark. Introduced an 'engine' setting in the
+  configuration that defaults to mapreduce, but can be set to 'spark'.
+
+- `CDAP-4128 <https://issues.cask.co/browse/CDAP-4128>`__ -
+  Added a new Javascript transform that can emit records using emitter. Have deprecated Script transform.
+
+- `CDAP-4251 <https://issues.cask.co/browse/CDAP-4251>`__ -
+  We removed the old fluent style api (deprecated two releases back). Developers should use the configurer
+  style API instead. AbstractFlow#configureFlow method has been deprecated. Use AbstractFlow#configure instead.
+
+- `CDAP-2303 <https://issues.cask.co/browse/CDAP-2303>`__ -
+  Added lookup capability in context that can be used in existing Script, ScriptFilter and Validator transforms.
+
+- `CDAP-4403 <https://issues.cask.co/browse/CDAP-4403>`__ -
+  Moved Hydrator plugins from the CDAP repository as cdap-etl-lib into its own repository.
+
+- `CDAP-4022 <https://issues.cask.co/browse/CDAP-4022>`__ -
+  Added RESTful APIs to start or stop multiple programs.
+
+- `CDAP-4023 <https://issues.cask.co/browse/CDAP-4023>`__ -
+  Added CLI commands to stop, start, restart, or get status of programs in an application.
+
+- `CDAP-4285 <https://issues.cask.co/browse/CDAP-4285>`__ -
+  Added an authorization plugin that uses a system dataset to manage ACLs
+
+- `CDAP-4235 <https://issues.cask.co/browse/CDAP-4235>`__ -
+  Added AuthorizationPlugin, for pluggable authorization.
+
+- `CDAP-4591 <https://issues.cask.co/browse/CDAP-4591>`__ -
+  Improvised Metadata Indexing and Search to support searches on words in value and tags.
+
+- `CDAP-4592 <https://issues.cask.co/browse/CDAP-4592>`__ -
+  Schema fields are not stored as Metadata and are searchable.
+
+Improvements
+------------
+- `CDAP-4209 <https://issues.cask.co/browse/CDAP-4209>`__ -
+  Added a method to ObjectMappedTable and ObjectStore to get a specific number of splits between a start and end key
+
+- `CDAP-4217 <https://issues.cask.co/browse/CDAP-4217>`__ -
+  Deprecated createDataSchedule and createTimeSchedule methods in Schedules class and removed
+  deprecated Schedule constructor.
+
+- `CDAP-3079 <https://issues.cask.co/browse/CDAP-3079>`__ -
+  Table datasets, and any other dataset that implements RecordWritable<StructuredRecord>,
+  can now be written to using Hive.
+
+- `CDAP-4444 <https://issues.cask.co/browse/CDAP-4444>`__ -
+  Make xmllint dependency optional and allow setting variables to skip configuration file parsing
+
+- `CDAP-4200 <https://issues.cask.co/browse/CDAP-4200>`__ -
+  The Serialization UID given by Java might be different if compiled with a different version of java or
```

Comment:
language is incorrect for release notes - 'this might not be an issue so closing this'. Perhaps just remove this item. I think in this case a comment was mistakenly added to the release notes field in jira.